cli/spec/support/output_helpers.rb

```diff
@@ -55,7 +55,16 @@ module OutputHelpers
         @real.zip(@expected) do |real, expected|
           line += 1
           fields = real.split(/\s{2,}/)
-          unless values_match?(fields, expected)
```

Comment:
Looks like there's a simpler way to support regexps here: per [rspec custom matchers supporting compound matchers](https://relishapp.com/rspec/rspec-expectations/v/3-5/docs/custom-matchers/define-a-custom-matcher#using-values-match?-to-compare-values-and/or-compound-matchers.), this just needs to be fixed to swap the argument order to `unless values_match?(expected, fields)`.

Then this works in the specs:

```ruby
    expect{subject.run([])}.to output_table [
      ['testvol',  'testscope', 'testdriver', a_string_matching(/\d+ days ago/)],
      ['testvol2', 'testscope2', 'testdriver2', a_string_matching(/\d+ days ago/)]
    ]
```

Reply:
nice